lib/Common/DataStructures/Comparer.h

```diff
@@ -112,19 +112,15 @@ struct RecyclerPointerComparer
     }
 };
 
-// FNV-1a hash -> https://en.wikipedia.org/wiki/Fowler%E2%80%93Noll%E2%80%93Vo_hash_function
-// #define CC_HASH_OFFSET_VALUE 2166136261
-// #define CC_HASH_LOGIC(hash, byte) \
-//    hash ^= byte;                  \
-//    hash *= 16777619
-
-// previous hash function.
-// TODO: hash function below is bad for key distribution.
```

Comment:
This hash function is indeed bad. Deserializer must do a lot of hashing and I saw bucket chains more than 10 deep. That is a lot of string comparisons.